eng/Version.Details.xml

```diff
@@ -42,6 +42,11 @@
       <Sha>27e584661980ee6d82c419a2a471ae505b7d122e</Sha>
       <SourceBuild RepoName="symreader" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.symreader" Version="2.1.0-beta.23409.1">
+      <Uri>https://github.com/dotnet/symreader</Uri>
+      <Sha>6f3128d255b0b1eb177712c7f2ab2ef0a147d251</Sha>
```

Comment:
This is adding incoherency to the dependencies.  

    <Dependency Name="Microsoft.SourceBuild.Intermediate.symreader" Version="2.0.0-beta-23228-04">
      <Uri>https://github.com/dotnet/symreader</Uri>
      <Sha>27e584661980ee6d82c419a2a471ae505b7d122e</Sha>